docs/src/content/docs/it/guides/sidebar.mdx

```diff
@@ -0,0 +1,480 @@
+---
+title: Navigazione nella barra laterale
+description: scopri come impostare e personalizzare i collegamenti di navigazione della barra laterale del tuo sito Starlight.
```

Comment:
```suggestion
description: Scopri come impostare e personalizzare i collegamenti di navigazione della barra laterale del tuo sito Starlight.
```

(not sure if it should be capitalized or not)

Reply:
it is in English. Will change.